fix weak parameter and add remove paths function

JackismyShephard opened the pull request:
This PR addresses https://github.com/zackees/static-sox/issues/2. Additionally, this PR also adds a `remove_from_path` function which removes the entries that `add_to_paths` adds to the `PATH` and `LD_PRELOAD` environment variables

JackismyShephard committed:
fix weak parameter and add remove paths function

zackees commented:
add comments on the LD preload stuff for remove paths. That's not obvious what you are doing.

Thanks for the PR!

JackismyShephard commented:
> add comments on the LD preload stuff for remove paths. That's not obvious what you are doing.
> 
> Thanks for the PR!

Will do !

JackismyShephard commented:
@zackees Comment has been added now. I also added a py.typed file to the root of the project. This is solely to appease mypy and pyright as they will otherwise complain that the file is missing. FYI I also added a PR with similar changes on https://github.com/zackees/static_ffmpeg

zackees commented:
This is now live, pip install -U static-sox to update
